examples/testing.pdl

```diff
@@ -0,0 +1,11 @@
+big_endian_packets
+
+enum Enum7 : 7 {
```

Comment:
Does this test file cover a specific case that is missed in the canonical test cases ?